migrations/2023-12-15-062816__net_amount_in_payment_attempt/up.sql

```diff
@@ -0,0 +1,8 @@
+
+ALTER TABLE payment_attempt
+ADD COLUMN IF NOT EXISTS net_amount BIGINT;
+
+UPDATE payment_attempt pa
+SET net_amount = pa.amount + COALESCE(pa.surcharge_amount, 0) + COALESCE(pa.tax_amount, 0);
```

Comment:
This migration should be run with caution in the higher environments, as this would lock the table. Instead can we have the above queries done in the following manner, to reduce the updates

```suggestion
ALTER TABLE payment_attempt
ADD COLUMN IF NOT EXISTS net_amount BIGINT NOT NULL DEFAULT 0;

UPDATE payment_attempt pa
SET net_amount = pa.amount + COALESCE(pa.surcharge_amount, 0) + COALESCE(pa.tax_amount, 0) WHERE pa.surcharge_amount > 0;
```